frontend/src/App.css

```diff
@@ -36,3 +42,36 @@
     transform: rotate(360deg);
   }
 }
+
+.card-container, .card-body, .login-form, .register-form, .form-group {
```

Comment:
Please read the "MUI React" docs. These styles are defined by the framework already. You don't need to specify these styles.

You can find some examples from the existing FE codes (Search for `Card` in FE codes).